### DISCUSSION TITLE
shaders文件过多导致的播放卡顿等

### DISCUSSION THREAD

**Discussion by iforgiveu** [hooke007/mpv_PlayKit#503] [Q&A 问答]
感觉是个bug，shaders的的glsl文件很多导致文件夹体积很大（七八十多m），尽管根本没有设置调用它们，但视频播放的画面会变得一卡一卡的。而当我把大部分的文件移走时，视频播放又恢复了流畅。
另外，大概24年起的版本的mpvlazy的菜单栏在高分辨率屏幕上缩放地很小，但是旧版本的菜单栏能保持合适看的大小，这是什么导致的呢

**Comment by hooke007** [comment]
1 无法复现，至少上传日志，否则无从谈起
2 我是4k屏没发现哪里小，可以自己改缩放，见脚本设置选项
